SQF/dayz_code/compile/A_Plot_for_Life/fn_check_owner_friends.sqf

```diff
@@ -49,7 +49,7 @@ if (DZE_permanentPlot) then {
 	_owner = (_targetOwner == dayz_characterID);
 };
 
-if (_targetOwner in _friendlies) then {
+if (_player in _friendlies) then {
```

Comment:
_player is an object and _friendlies is a list of strings, so this will always return false. To call this function with door management you need to merge in a separate check of the "doorfriends" variable similar to how the check of the "plotfriends" works above. If this is beyond your ability don't worry about it. Just revert this change and I'll add it in later.

I am aware Plot Management has no access control atm. Here in fn_selfActions:
https://github.com/EpochModTeam/DayZ-Epoch/blob/master/SQF/dayz_code/compile/fn_selfActions.sqf#L635

No condition is checked before the s_player_plotManagement action is added. It should be inside the IF statement directly above it. It looks like that was a mistake by icomrade when he merged A Plot for Life.